### DISCUSSION TITLE
Was the older spectrum ariadne history lost ?

### DISCUSSION THREAD

**Discussion by kleysonr** [mirumee/ariadne#536] [General]
I'm trying to access the older spectrum ariadne chat looking for some posts there, but it's no longer available anymore.

Was the whole spectrum history lost ?

**Comment by rafalp** [comment]
Pretty much, yes. Spectrum had feature that claimed to make community read-only and display link to new community, but turns out that wasn't what it said on the tin.

If you have any questions, please ask them there. I'm little tight on time at the moment, but I'll try to answer.

**Reply by kleysonr** [reply]
@rafalp A so rich source of information and knowledge, lost !!!

So many discussions, sample codes and snippets.

Is there no way to bring it back to life ?

I can't believe that is all lost :(

**Comment by kleysonr** [comment]
@rafalp Looks like all the post are still there.

--> https://spectrum.chat/ariadne/general/help-to-define-a-type-where-a-field-is-an-array-of-objects~a74dd164-c69c-4689-a506-6859c38e52f1

Please, make them available again.

Best Regards.

**Comment by rafalp** [comment]
I'll try to reach out to Spectrum folk about this. They advertised that migrating community will mark it as read-only and will add banner pointing to new site, but this is not the case.
